### ISSUE TITLE
Improve visible focus in `tile` components

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#2459]
Scope
---
Describe clearly the scope of the task.

Out of Scope
---
List topics that are not part of this task.

Notes  
---
Community feedback: https://porschedev.slack.com/archives/C035YG03Q02/p1680770973870529

Acceptance criteria
---
- [ ] Criteria

Subtasks
---
- [ ] Task

**Comment by Lezette** [comment]
@o-hook is this issue still open I'll like to work on it

**Comment by stephanschroeter** [comment]
Will be refactored entirely with a generic tile component.
